cmd/tdexdconnect/main.go

```diff
@@ -170,7 +176,7 @@ func main() {
 	}
 
 	connectUrl, err := tdexdconnect.EncodeToString(
-		rpcServerAddr, certBytes, macBytes,
+		schema, rpcServerAddr, certBytes, macBytes,
```

Comment:
shoudn't here be used the ConnectUrlHost instead of rpcserver address?

Reply:
the tdexdconnect binary does not have access to daemon's configuration.. rather, the daemon should use it to build the correct `rpcServerAddr`.

Reply:
Yes what I mean is inside tdexd where we generate the HTML page to `rpcServerAddr` we must pass the connect URL, not the OPERATOR_LISTENING_PORT